global_helpers/panther_base_helpers.py

```diff
@@ -309,6 +310,60 @@ def deep_get(dictionary: dict, *keys, default=None):
     return out
 
 
+# pylint: disable=R1260,R0911
+def deep_walk(obj: dict, *keys: str, default: str = None, return_val: str = "all") -> Optional[str]:
```

Comment:
it seem to me  the type hint for deep_walk's return value should be something like `-> Optional[str, List[str]]`

Reply:
I took it out but I agree that re-adding a more accurate type hint would be preferable.

Reply:
Addressed in [`d7b2e68` (#819)](https://github.com/panther-labs/panther-analysis/pull/819/commits/d7b2e68d4cf61894708d48c679908da41b9a5b00).